__tests__/columncrud.specs.js

```diff
@@ -0,0 +1,160 @@
+import '@babel/polyfill';
+import '../src/utils/dotenv';
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import chaiAsPromised from 'chai-as-promised';
+import JWT from 'jsonwebtoken';
+import { server } from '../src/app';
+import { User, Board, Column } from '../src/models';
+
+chai.use(chaiAsPromised);
+
+const { expect } = chai;
+
+chai.use(chaiHttp);
+
+
+describe('Column', () => {
+  describe('/GET column', () => {
+    const user = new User({
+      displayName: 'Zalupa',
+      googleId: 'kalsndlk123lk',
+      nickname: 'piska',
+    });
+    user.save();
+
+    const board = new Board({
+      authorId: user._id,
+      columns: [],
+    });
+    board.save();
+
+    const column = new Column({
+      name: 'BOY',
+      authorId: board._id,
+      tasks: [],
+    });
+    column.save();
```

Comment:
Why do you add column to the board, if you don't check if it exists in response?